src/main/webapp/SearchResults.js

```diff
@@ -1,18 +1,32 @@
 /**
  * @param {query} input
  */
-function searching(input) {
+async function searching(input) {
   input = input.toLowerCase();
+  var countryId = null;
+  var countryName = null;
+
+  const countriesResponse = await fetch('/api/countries');
+  const countries = await countriesResponse.json();
+  
+  for (let country in countries) {
+    if(countries[country]['name'].toLowerCase() === input){
+        displayCountryResults(countries[country]);
+        countryId = countries[country]['id'];
+        countryName = countries[country]['name'];
+        break;
+    }
+  }
+
```

Comment:
This looks like it's only looking for a single country with the exact name `input`. Don't we want to show multiple results for non-exact country names? E.g. the query `Pa` might country results for both Pakistan and Panama? You could use [`string.includes(searchvalue)`](https://www.w3schools.com/jsref/jsref_includes.asp) for this.